data/reserved_words_mysql.txt

```diff
@@ -497,7 +504,6 @@ relay_thread
 relaylog
 release
 reload
-remote
```

Comment:
Not present in [MySQL 8.0 Removed Keywords and Reserved Words](https://dev.mysql.com/doc/refman/8.0/en/keywords.html#keywords-removed-in-current-series),
but added in 8.0.3 and removed in 8.0.14.